spec/fixtures/files/peer_reviews/.gitignore

```diff
@@ -0,0 +1 @@
+temp_peer_review.csv
```

Comment:
Why don't we make the test remove the file if it exists after it's done executing.